toolchain/check/testdata/function/builtin/types.carbon

```diff
@@ -0,0 +1,203 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
```

Comment:
`check/testdata/function/builtin` is for tests checking our handling of function declarations with builtin definitions in general, not for the behavior of specific builtins. For specific builtins, I've been adding one file per builtin to `check/testdata/builtins/<name>` (with `.`s in the name replaced by `/`s).

Reply:
Done